classes/PodsUI.php

```diff
@@ -4927,16 +4927,43 @@ public function restricted( $action = 'edit', $row = null ) {
 
 				if ( pods_is_admin( array( 'pods', 'pods_content' ) ) ) {
 					$restricted = false;
-				} elseif ( 'manage' === $action ) {
-					if ( ! in_array( 'edit', $this->actions_disabled ) && ( current_user_can( 'pods_edit_' . $this->pod->pod ) || current_user_can( 'pods_edit_others_' . $this->pod->pod ) ) ) {
-						$restricted = false;
-					} elseif ( ! in_array( 'delete', $this->actions_disabled ) && ( current_user_can( 'pods_delete_' . $this->pod->pod ) || current_user_can( 'pods_delete_others_' . $this->pod->pod ) ) ) {
-						$restricted = false;
-					} elseif ( current_user_can( 'pods_' . $action . '_' . $this->pod->pod ) || current_user_can( 'pods_' . $action . '_others_' . $this->pod->pod ) ) {
-						$restricted = false;
+				} else {
+
+					// Check if the current user is the author of this item.
+					$author = pods( $this->pod->pod, $row )->field( 'author', true );
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 4 spaces but found 1 space

<sub>hash: ddfd2bf3441648bbe8c5ca120f39821f</sub>